api/bsf_handler.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// apiBSFHandler fetches browser-specific failure data based on the URL params.
+func apiBSFHandler(w http.ResponseWriter, r *http.Request) {
+	handleBSF(w, r, shared.NewFetchBSF())
+}
+
+func handleBSF(w http.ResponseWriter, r *http.Request, fetcher shared.FetchBSF) {
+	q := r.URL.Query()
+	isExperimental := false
+	val, _ := shared.ParseBooleanParam(q, "experimental")
+	// If the experimental parameter is missing or present with no value, set
+	// isExperimental to the default value, false.
```

Comment:
This comment is slightly confusing as it's not what the code actually does (though with the same effect). Maybe just drop it.